techlibs/xilinx/synth_xilinx.cc

```diff
@@ -82,10 +97,10 @@ struct SynthXilinxPass : public ScriptPass
 		log("        do not use XORCY/MUXCY/CARRY4 cells in output netlist\n");
 		log("\n");
 		log("    -nowidelut\n");
-		log("        do not use MUXF[78] resources to implement LUTs larger than LUT6s\n");
+		log("        do not use MUXF[56789] resources to implement LUTs larger than LUT4s/LUT6s\n");
```

Comment:
I feel like `do not use MUXF[56789] resources to implement LUTs larger than native to hardware` is less confusing.

Reply:
Or `do not use MUXF[56789] resources to implement LUTs larger than supported natively`?